e2e_playwright/st_data_editor_column_types_test.py

```diff
@@ -0,0 +1,28 @@
+# Copyright (c) Streamlit Inc. (2018-2022) Snowflake Inc. (2022)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from playwright.sync_api import Page, expect
+
+from e2e_playwright.conftest import ImageCompareFunction
+
+
+def test_data_editor_column_types(
+    themed_app: Page, assert_snapshot: ImageCompareFunction
+):
+    """Test that st.data_editor render various column types correctly."""
+    data_editor_elements = themed_app.locator(".stDataFrame")
```

Comment:
nit: it's slightly preferred to use the testid instead if available:

```suggestion
    data_editor_elements = themed_app.get_by_test_id("stDataFrame")
```
This also applies to the other playwright tests in this PR. I just recently started doing that, so most of the "older" playwright tests are still using classname.

Reply:
Ah thanks. Good to know.